src/qibolab/tests/test_transpilers_native.py

```diff
@@ -22,7 +22,7 @@ def assert_matrices_allclose(gate):
     native_unitary = native_matrix / np.power(
         np.linalg.det(native_matrix), 1 / float(native_matrix.shape[0]), dtype=complex
     )
-    np.testing.assert_allclose(native_unitary, target_unitary, atol=1e-12)
+    np.testing.assert_allclose(phase * native_unitary, target_unitary, atol=1e-12)
```

Comment:
Shouldn't this phase be removed by the det normalization?

Reply:
Unfortunately if the determinant of the matrix is +1 and the difference global phase is pi there may be problems, this happened for the FSWAP gate

Reply:
Thanks.